bundles/nl.esi.comma.testspecification/src/nl/esi/comma/testspecification/GenerateTestspecification.mwe2

```diff
@@ -33,7 +33,6 @@ Workflow {
 				encoding = "UTF-8"
 				lineDelimiter = "\r\n"
 				fileHeader = "/*\n * generated by Xtext \${version}\n */"
-				preferXtendStubs = false
```

Comment:
> It looks like the pull request has much more edits than the name of the branch suggests. I had a look at the merging part and that seems okay but would need to test it out with the reference model as a last sanity check. We could also do this after merging into main. Feel free to propose.

@dbera, yes ,that may look that way, but most were caused due to this small change. I could have commented on that. In order to write a validation, I wanted to be able to use Xtend, and this was the only language for which Xtend stubs were not enabled yet.